core/cont/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ endif()
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/res ${CMAKE_CURRENT_SOURCE_DIR}/../foundation/res)
 
-ROOT_GLOB_HEADERS(headers ${CMAKE_CURRENT_SOURCE_DIR}/inc/*.h)
+ROOT_GLOB_HEADERS(headers ${CMAKE_CURRENT_SOURCE_DIR}/inc/*.h ${CMAKE_CURRENT_SOURCE_DIR}/inc/*.hxx)
```

Comment:
I think you meant ```inc/ROOT/*.hxx```

Reply:
Not sure what the policy is. `.hxx` are always in `inc/ROOT/`?

Reply:
yes  ... and at the very least the header targeted here *is* in inc/ROOT :)

Reply:
Ah true, the globbing isn't recursive.